helper-images/dummyio/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.11-slim as builder
+
+RUN python -m venv .venv
+COPY requirements.txt .
+RUN .venv/bin/pip install -r requirements.txt
+
+FROM python:3.11-slim
+
+COPY --from=builder /.venv /.venv
```

Comment:
Why do we even need staged builds if we copy the entire venv?

Reply:
Probably more a habit than anything else.
The only valuable point with such an easy image is that we don't keep requirements.txt in the final image.
Which might also be achieved with a `RUN rm`.
Does this bother us?

Reply:
>  Which might also be achieved with a RUN rm.

`RUN rm` will only make the image bigger, though

Not a blocker, ok to leave.